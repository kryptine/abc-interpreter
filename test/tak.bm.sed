s/24 16 8/36 24 12/
