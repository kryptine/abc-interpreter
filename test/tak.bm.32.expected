9000
halt at 6
9 524279 524288
